src/oc/storage/db/migrations/1534797688142_entry_versions.edn

```diff
@@ -0,0 +1,22 @@
+(ns oc.storage.db.migrations.entry-versions
+  (:require [rethinkdb.query :as r]
+            [oc.lib.db.migrations :as m]
+            [oc.storage.config :as config]
+            [oc.storage.resources.entry :as entry]))
+
+(defn up [conn]
+
+  (println "Creating table: " entry/versions-table-name)
+  (println (m/create-table conn config/db-name entry/versions-table-name entry/versions-primary-key))
+  (println (m/create-index conn entry/versions-table-name "revision-id"))
+  (println (m/create-index conn entry/versions-table-name "topic-slug"))
```

Comment:
I think `topic-slug` is an old deprecated concept. Did you add that here just to mirror the `entries` table?